File: 2 SEGUNDO/1 Cuatrimestre/Programación y Diseño Orientado a Objetos - PDOO/Practicas/javaNapakalaki/NapakalakiGame/DeathBadConsequence.java
package NapakalakiGame;

/**
 *
 * @author manjaro
 */

public class DeathBadConsequence extends NumericBadConsequence {

    private boolean death;

    public DeathBadConsequence(String text, boolean death) {
        super(text, 0, 0, 0);
        this.death = death;
    }

    @Override
    public boolean isDeath() {
        return death;
    }

}

File: 2 SEGUNDO/1 Cuatrimestre/Programación y Diseño Orientado a Objetos - PDOO/Practicas/rubyNapakalaki/death_bad_consequence.rb
class DeathBadConsequence < NumericBadConsequence

  attr_accessor :death


  def initialize(text)
    super(text,0,0,0)
    @death = true
  end

  def adjust_to_fit_treasure_list(v, h)
      bad_cons = DeathBadConsequence.new('')
      bad_cons
  end

end
